app/src/main/java/com/example/candidatecodereview/MainFragment.kt

```diff
@@ -0,0 +1,44 @@
+package com.example.candidatecodereview
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import kotlinx.android.synthetic.main.fragment_main.*
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.MainScope
+import kotlinx.coroutines.launch
+
+class MainFragment : Fragment() {
+
+    val listener: LiveData<Int>
+        get() = mutableListener
+    private val mutableListener = MutableLiveData<Int>()
```

Comment:
I would rename `listener` to something like `toolbarTitleListener` and `mutableListener` to `mutableToolbarTitleListener`.